src/main/java/frc/trigon/robot/subsystems/ledstrip/LEDStripCommands.java

```diff
@@ -0,0 +1,60 @@
+package frc.trigon.robot.subsystems.ledstrip;
+
+import com.ctre.phoenix.led.ColorFlowAnimation;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+
+public class LEDStripCommands {
+    public static Command getSetLEDs(Color colour, LEDStrip ledStrip) {
```

Comment:
All of the commands in this class can and should be start end commands
There's no need to clear animations before applying new ones (apart from the setLEDs command) 
also, get...Command